makefile.shade

```diff
@@ -5,16 +5,6 @@ k-standard-goals
 
 default DOTNET_ARGUMENT='-v '
 
-#build-compile target='compile'
```

Comment:
> We want to remove as much as possible from makefiles.

We do not want experimental projects built by default, ever. The idea here is to provide an _additional_ target that handles those builds when needed. Please restore this target.